### ISSUE TITLE
Error in computing aggregate investment

### ISSUE THREAD

**Issue by jdebacker** [open-source-economics/OG-USA#412]
In the case where `small_open = True`, we compute aggregate investment as (lines 1023-1024 of `TPI.py`):

```
 I = ((1 + g_n_vector[:T]) * np.exp(g_y) * K[1:T + 1] -
             (1.0 - delta) * K[:T])
```

I believe the indexing for `g_n_vector` is off here and that the equations should read:

```
 I = ((1 + g_n_vector[1:T + 1]) * np.exp(g_y) * K[1:T + 1] -
             (1.0 - delta) * K[:T])
```

This would make the equation consistent with Equation A.1.32 of [DeBacker, Evans, and Phillips (Public Finance Review, forthcoming)](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3079198).

Note that there is no check on this equation since it does not enter into the resource constraint.

**Comment by jdebacker** [comment]
@rickecon Can you confirm that the indexing is indeed off in the code in this case?

**Comment by jdebacker** [comment]
Resolved in PR #423.
